go/src/koding/kites/kontrol/kontrol.go

```diff
@@ -51,5 +53,9 @@ func main() {
 		k.Kite.Log.Error(err.Error())
 	}()
 
+	if os.Getenv("GENERATE_DATADOG_DASHBOARD") != "" {
```

Comment:
At first i have implemented it like that but it leaves the impression that it will create the dashboard each time that function executes - but in fact it isn't, so i made it implicit.